notification/twilio/alertsms.go

```diff
@@ -89,14 +89,34 @@ func mapGSM(r rune) rune {
 	return '?'
 }
 
+func oneOfPrefixes(s string, prefixes ...string) bool {
+	for _, p := range prefixes {
+		if strings.HasPrefix(s, p) {
+			return true
+		}
+	}
+	return false
+}
+
+// canContainURL returns true if the message can contain a URL.
+func canContainURL(ctx context.Context, number string) bool {
+	if config.FromContext(ctx).General.DisableSMSLinks {
+		return false
+	}
+
+	// not extensive, but CN code numbers do not allow URLs in SMS
+	// https://www.twilio.com/guidelines/cn/sms
+	return !oneOfPrefixes(number, "+86")
+}
+
 // hasTwoWaySMSSupport returns true if a number supports 2-way SMS messaging (replies).
 func hasTwoWaySMSSupport(ctx context.Context, number string) bool {
 	if config.FromContext(ctx).Twilio.DisableTwoWaySMS {
 		return false
 	}
 
-	// India numbers do not support SMS replies.
-	return !strings.HasPrefix(number, "+91")
+	// not extensive
+	return !oneOfPrefixes(number, "+91", "+86", "+502", "+506", "+507", "+84")
```

Comment:
could add some constants for these so that it's clearer what countries don't allow two way support, so you could do 
```
return !oneOfPrefixes(number, codes.China, codes.India, ...)
```

Reply:
Similar to above, constants would add noise and indirection here that would take away from readability and add mental overhead; none of these dialing codes really have any value outside of the func here.

We don't have a _current_ use case for mapping countries to dialing codes, (or region names to dialing codes for that matter), though we could feasibly map them through `libphonenumber` but again, probably not worth the cost of constantly mapping to-from region codes vs. just listing the prefix which can be directly compared to the contact method value.

I do think you're right about adding clarity around which regions these represent, so I added some better comments for quick reference to the next/future readers.